extension-requests/script.js

```diff
@@ -202,22 +213,17 @@ function getCheckedValues(groupName) {
 applyFilterButton.addEventListener('click', async () => {
   filterModal.classList.toggle('hidden');
   const checkedValuesStatus = getCheckedValues('status-filter');
-  statusChange();
-  await populateExtensionRequests({
-    status: checkedValuesStatus,
-    size: 10,
-    dev: params.get('dev') === 'true',
-  });
+  filterChange();
+  updateFilterStates('status', checkedValuesStatus);
```

Comment:
- will `updateFilterStatus` be a better name here?

Reply:
No, we are also storing other states like `order` of the cards and in future assignee id.